src/CMakeLists.txt

```diff
@@ -65,6 +66,9 @@ if(NGRAPH_TF_ENABLE_VARIABLES_AND_OPTIMIZERS)
     list(REMOVE_ITEM SRC ngraph_rewrite_pass.cc)
     list(APPEND SRC enable_variable_ops/ngraph_rewrite_pass.cc)
 
+    list(REMOVE_ITEM SRC ops/ngraph_ops.cc)
+    list(APPEND SRC enable_variable_ops/ops/ngraph_ops.cc)
```

Comment:
We have some registerations here too, if you want to take care of them too
https://github.com/tensorflow/ngraph-bridge/blob/master/src/enable_variable_ops/ngraph_variable_modifiers.cc